tests/openssl_moretest/host/digest_tests.cpp

```diff
@@ -0,0 +1,392 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/internal/tests.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+//#include "common.h"
+#include "CSchema_checker.h"
+#include "include_openssl.h"
+#include "openssl/e_os2.h"
```

Comment:
Assume you update the inclusion as I suggested, you can modify this:

```suggestion
#include <openssl/e_os2.h>
```